langchain/src/retrievers/self_query/tests/vectara_self_query.init.test.ts

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-process-env */
+import { test } from "@jest/globals";
+import { Document } from "../../../document.js";
+import { AttributeInfo } from "../../../schema/query_constructor.js";
+import { SelfQueryRetriever } from "../index.js";
+import { OpenAI } from "../../../llms/openai.js";
+import { VectaraTranslator } from "../vectara.js";
+import { FakeEmbeddings } from "../../../embeddings/fake.js";
+import { VectaraStore } from "../../../vectorstores/vectara.js";
+
+test.skip("Vectara Self Query Retriever Test", async () => {
```

Comment:
Shouldn't this be test() and not test.skip()

Reply:
Since this test requires the credentials and account setup of vectara thats why it is skipped. if we don't skip it would fail whenever test suit runs. Other vector stores are also using the `test.skip` .

Reply:
Yes you are right. Good point.